app/(pages)/admin/projects/approval/page.tsx

```
@@ -655,424 +691,555 @@ export default function ProjectApprovalPage() {
+                        <h4 className="font-semibold mb-3 flex items-center space-x-2">
+                          <Video className="w-4 h-4" />
+                          <span>Project Video</span>
+                        </h4>
+                        <div className="relative aspect-video border rounded-lg overflow-hidden bg-gray-100">
+                          <video
+                            src={getUrl(selectedProject.media.video)}
+                            controls
+                            className="w-full h-full"
+                          >
+                            Your browser does not support the video tag.
+                          </video>
```

Biome (2.4.4)

[error] Provide a track for captions when using audio or video elements.

(lint/a11y/useMediaCaption)